### DISCUSSION TITLE
Testing framework preferences.

### DISCUSSION THREAD

**Discussion by Micosilent** [TriliumNext/Notes#39] [Q&A / Support]
Simply creating this discussion to gather information about preferences in testing frameworks. 

If there isn't any strong objections I will start work on setting up and implementing some test with Jest via ts-jest over the weekend.

**Comment by zerebos** [comment]
Not a super strong preference but I usually use Mocha + Chai. I have worked with Ava before as well.

**Comment by meichthys** [comment]
Related to #19
